elections/tac-and-scir-election-process.md

```diff
@@ -0,0 +1,65 @@
+
+
+## Voter Eligibility (Electorate) Self-Nomination Process
+
+Any contributor to OpenSSF working groups, SIGs or projects is eligible to participate in the election.
+Valid contributions include: commits or submitted pull requests via Github; public edits or comments on Google docs or other work products associated with OpenSSF; posting messages to any mailing list or on Slack; and beyond that any other form of positive engagement with OpenSSF activities.
+The form asks you for an example of your contributions; this is merely to make it easier for the Election Officials and OpenSSF staff to validate eligability.
+If you have in any way been involved in or care about OpenSSF, but are in doubt as to whether your contribution “counts”, please fill it out anyways, and we will follow up.
+
+[Voter Eligibility Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSegUTOrxrGPVwDrPH9RARmYxX5YEkW3j67RU4uWABkKD5x8og/viewform)
+
+ 
+## TAC Self-Nomination Process
+
+The OpenSSF Technical Advisory Council (TAC) is composed of seven total individuals, four of whom are elected annually.
+Eligable community members, according to the criteria defined above, can self-nominate using the form below.
+
+[TAC Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSdxUbbfJGgjU_cZ_uGvUm-5ePoXu7q2pGiNyH2G8euwpn23Sw/viewform)
+
+
+## SCIR Self-Nomination Information and Process
+
+Since early in its existence, the OpenSSF Governing Board has sought to ensure it gets adequate input from voices in the software security community who would otherwise not be at the table.
+We seek candidates for the Security Community Individual Representative (SCIR) who can represent those voices, while also being a subject matter expert in the field with their own set of perspectives.
+Familiarity with the different OpenSSF working groups and projects, and being able to dedicate the time to be sufficiently informed on the issues that arise in our monthly calls and ongoing discussions, is highly desired.
+It is also highly desired, but not required, that the SCIR be a contributor and thus eligible to vote in the election.
+
+[SCIR Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLScoabVVjW1kvTNcKXhTEdvxQd13Caql-I36MOixHVO1dLwIiQ/viewform)
+
+
+## Election Process
+
+The Election Process will be monitored by Election Officials, who
+* review nominees and confirm their eligibility,
+* approve the election timeline, and
+* review the ballot request form and validate voter eligibility.
+
+Election Officials are eligable community members who are not running for a TAC or SCIR position, but may cast a vote.
+
+During the validation period, the Election Officials will review the pool of eligible voters to confirm eligibility to participate in the election, per the guidelines defined above.
+At the conclusion of the validation period, Election Officials will publish the complete list of candidates.
```

Comment:
Good point. As during last year's election, the list of candidates should be published in this directory of this repo.

I don't mind publishing the list of eligable voters (just their names) as well.

In case a ballot requestor is considered not eligible, the Election Officials should notify this person.

Reply:
Updated to reflect 👍